frontend/src/layout/onboarding/onboarding.js

```diff
@@ -0,0 +1,162 @@
+    const { user } = useValues(userLogic)
+    const { loadUser } = useActions(userLogic)
+    const { tourType, checked } = useValues(onboardingLogic({ user }))
+    const { setTourActive, setTourType, updateOnboardingInitial } = useActions(onboardingLogic)
+    const [visible, setVisible] = useState(user.onboarding.initial ? true : false)
+
+    const unfinishedCount = _.filter(checked, (isChecked) => !isChecked).length
+
+    function closePopup() {
+        if (user.onboarding.initial) updateOnboardingInitial(false)
+        setVisible(false)
+    }
+
+    let onClickOutside = (event) => {
+        if (contentRef.current && !contentRef.current.contains(event.target)) {
+            closePopup()
+        }
+    }
+
+    useEffect(() => {
+        document.addEventListener('mousedown', onClickOutside)
+        return () => {
+            document.removeEventListener('mousedown', onClickOutside)
+        }
+    }, [])
+
+    async function dontShowAgain() {
+        await api.update('api/user', { onboarding: { ...user.onboarding, active: false } })
+        loadUser()
+    }
+
+    function content() {
+        return (
+            <div
+                data-attr="onboarding-content"
+                ref={contentRef}
+                style={{ display: 'flex', width: '25vw', flexDirection: 'column' }}
+            >
+                <h2>Get Started</h2>
+                <i>
+                    Complete these steps to learn how to use Posthog! Click on an item below to learn how to create one
+                </i>
+                {Object.entries(TourType).map(([, value], index) => {
+                    return (
+                        <div key={index}>
+                            <hr style={{ height: 3, visibility: 'hidden' }} />
+                            <Link
+                                onClick={() => {
+                                    closePopup()
+                                    setInstructionalModal(true)
+                                    setTourType(value)
+                                }}
+                                data-attr={'onboarding-item-' + index}
+                            >
+                                <Checkbox
+                                    style={{ marginRight: 12 }}
+                                    checked={user.onboarding.steps[index] || checked[index]}
+                                ></Checkbox>
+                                <span>{value}</span>
+                            </Link>
+                        </div>
+                    )
+                })}
+                <hr style={{ height: 5, visibility: 'hidden' }} />
+                {unfinishedCount > 0 ? (
+                    <p onClick={dontShowAgain} style={{ color: 'gray', cursor: 'pointer' }}>
+                        Don't show this again
+                    </p>
+                ) : (
+                    <Button onClick={dontShowAgain} type={'primary'}>
```

Comment:
If this is a common scenario, you might want to create a `PrimaryButton`
```
function PrimaryButton(props) {
  return <Button {...props} type={'primary'} />
}
```

Another thing is, setting `style` everywhere is annoying and verbose. With the above model you can even do this:
```
function PrimaryButton({ color, style, ...rest }) {
  return <Button style={{color, ...style}} {...rest} type={'primary'} />
}
```
That way you can do `<PrimaryButton color="red" />` which is much more concise than `<Button style={{ color: "red" }}`

(StyledSystems already supports this more concise style)